code/controllers/subsystems/traumas.dm

```diff
@@ -0,0 +1,125 @@
+												 /obj/item/clothing/head/helmet/space/void/captain, /obj/item/clothing/suit/space/void/captain, /obj/item/clothing/suit/storage/hooded/wintercoat/captain,
+												 /obj/item/clothing/under/captainformal, /obj/item/clothing/under/rank/captain, /obj/item/toy/figure/captain, /obj/item/weapon/card/id/captains_spare,
+												 /obj/item/weapon/gun/energy/captain, /obj/item/weapon/stamp/captain, /obj/item/weapon/storage/backpack/captain, /obj/item/clothing/suit/armor/vest/security)),
+
+					   "filth"   = typecacheof(list(/obj/effect/decal/cleanable,/obj/effect/decal/remains,/obj/item/trash, /obj/machinery/portable_atmospherics/hydroponics)),
+
+					   "paranormals"   = typecacheof(list(/obj/item/weapon/book/tome,/obj/item/weapon/nullrod,/obj/structure/constructshell,
+												/obj/structure/cult, /obj/structure/girder/cult, /obj/structure/grille/cult, /obj/effect/forcefield/cult,
+												/obj/effect/gateway, /obj/item/clothing/head/culthood, /obj/item/clothing/head/helmet/space/cult, /obj/item/clothing/shoes/cult,
+												/obj/item/clothing/suit/cultrobes, /obj/item/clothing/suit/space/cult, /obj/item/toy/cultsword, /obj/item/weapon/melee/cultblade,
+												/obj/item/weapon/storage/backpack/cultpack, /obj/item/weapon/storage/bible, /obj/effect/rune)),
+
+					   "doctors" = typecacheof(list(/obj/item/weapon/gun/launcher/syringe, /obj/item/weapon/reagent_containers/syringe, /obj/item/clothing/suit/storage/toggle/labcoat,
+												 /obj/item/toy/figure/md, /obj/item/weapon/bedsheet/medical, /obj/item/weapon/rig/medical, /obj/item/weapon/storage/backpack/duffel/med,
+												 /obj/item/weapon/storage/backpack/medic, /obj/item/weapon/storage/backpack/messenger/med, /obj/item/weapon/storage/belt/medical,
+												 /obj/machinery/vending/medical, /obj/mecha/medical/odysseus, /obj/structure/closet/crate/medical, /obj/structure/closet/medical_wall,
+												 /obj/structure/sign/greencross, /obj/item/clothing/accessory/armband/med, /obj/item/clothing/head/helmet/space/void/medical, /obj/item/clothing/mask/breath/medical,
+												 /obj/item/clothing/under/rank/medical, /obj/item/weapon/scalpel, /obj/machinery/clonepod, /obj/item/weapon/storage/firstaid, /obj/machinery/optable,
+												 /obj/item/clothing/accessory/armband/science, /obj/item/clothing/under/rank/scientist, /obj/machinery/door/airlock/science, /obj/machinery/door/airlock/medical))
+
+					   )
+	phobia_turfs = list("space" = typecacheof(list(/turf/space, /turf/simulated/open)),
+						"paranormals" = typecacheof(list(/turf/simulated/wall/cult, /turf/simulated/floor/cult)),
+						"filth" = typecacheof(list(/turf/simulated/floor/grass, /turf/simulated/mineral)),
+						"darkness" = typecacheof(list(/turf/unsimulated/mask)) //fuck this is so dangerously snowflake lohikar is going to eat my ass
+						)
+
+	phobia_species = list("lizards"   = typecacheof(list(/datum/species/unathi)),
```

Comment:
There are actual lizards that spawn frequently. Those seem like a better alternative.

Reply:
An alternative to what? Those are already included in the phobia.

Reply:
Correction; noticed the lizard smob's are under the wrong phobia.

Reply:
Hadn't seen that yet, I guess that works.